packages/Heading/src/Heading.styles.js

```diff
@@ -14,64 +16,75 @@ const headingStyles = () => `
     outline: none;
   }
 
-  &.heading--level-1 {
+  .heading__divider {
+    border-bottom: 2px solid ${tokens.color.blackLighten70};
+    flex: 1;
+    margin-left: ${tokens.space};
+  }
+`;
+
+// heading styles
+
+const heading = {
+  HEADING1: `
     ${stylers.fontSize(7)}
     ${stylers.lineHeight(-2)}
-    font-weight: 400;
-  }
+    font-weight: 400;`,
 
-  &.heading--level-2 {
+  HEADING2: `
     ${stylers.fontSize(5)}
-    ${stylers.lineHeight(-2)}
-  }
+    ${stylers.lineHeight(-2)}`,
 
-  &.heading--level-3 {
-    ${stylers.fontSize(3)}
-  }
+  HEADING3: `
+     ${stylers.fontSize(3)}`,
 
-  &.heading--level-4 {
-    ${stylers.fontSize(1)}
-  }
+  HEADING4: `
+     ${stylers.fontSize(1)}`,
 
-  &.heading--level-5 {
+  HEADING5: `
     ${stylers.fontSize()}
-    margin: ${tokens.space} 0;
-  }
+    margin: ${tokens.space} 0;`,
 
-  &.heading--level-6 {
+  HEADING6: `
     ${stylers.fontSize(-1)}
     margin: ${tokens.space} 0;
-  }
+  }`,
+};
 
-  &.heading--has-underline {
+// underline
+const underline = `
     border-bottom: 1px solid ${tokens.border.color};
-    &.heading--level-1 {
+    h1 {
```

Comment:
We can't use these because we have a prop that is `isSemantic`, which when false does not use . h1 to h6. It uses <div> to avoid styles override by global css instead.

Reply:
done